### ISSUE TITLE
Accessibility analysis should be completed and implemented

### ISSUE THREAD

**Issue by freelawbot** [freelawproject/courtlistener#179]
There's a good resource here, which essentially spells out what needs to be done. 

http://diveintoaccessibility.org/table_of_contents.html

It's all pretty basic stuff from what I've seen so far, and is a great place to get a feel for the basic templates.

Tags: easy-bug, accessibility, html, css, templates

---
- Bitbucket: https://bitbucket.org/mlissner/search-and-awareness-platform-courtlistener/issue/142
- Originally Reported By: [Mike Lissner](http://bitbucket.org/mlissner)
- Originally Created At: 2010-10-29T06:08:07.559
